data/translations/zh/state-data.json

```diff
@@ -3,311 +3,411 @@
     "hp_link": "https://www.sos.alabama.gov/alabama-votes",
     "registration_link": "https://www.alabamainteractive.org/sos/voter_registration/voterRegistrationWelcome.action",
     "more_info_link": "https://www.sos.alabama.gov/alabama-votes/voter/register-to-vote",
-    "ip_deadline": "選舉日前 15 天",
-    "online_deadline": "選舉日前 15 天",
-    "mail_postmarked_deadline": "選舉當日前 15 天郵戳為憑",
-    "confirm_registration_link": "https://myinfo.alabamavotes.gov/VoterView/RegistrantSearch.do"
+    "ip_deadline": "2022年10月24日星期六",
+    "online_deadline": "2022年10月24日星期六",
+    "mail_postmarked_deadline": "郵戳為憑（2022年10月24日星期六)",
```

Comment:
We need to remove the () from postmarked and received deadlines. We only have the translators do this so we can identify the start and end of the date in case we need to modify it later.

Reply:
fixed